package-build.el

```diff
@@ -1123,12 +1125,12 @@ Returns the archive entry for the package."
      ((not version)
       (error "Unable to check out repository for %s" package-name))
      ((= 1 (length files))
-      (pb/build-single-file-package package-name version (caar files) source-dir target-dir))
+      (pb/build-single-file-package package-name version (caar files) source-dir target-dir dependencies))
      ((< 1 (length  files))
-      (pb/build-multi-file-package package-name version files source-dir target-dir))
+      (pb/build-multi-file-package package-name version files source-dir target-dir dependencies))
      (t (error "Unable to find files matching recipe patterns")))))
 
-(defun pb/build-single-file-package (package-name version file source-dir target-dir)
+(defun pb/build-single-file-package (package-name version file source-dir target-dir dependencies)
```

Comment:
The problem with replacing is that if `dependencies` is nil, does that mean "leave the existing dependencies" or "there are no dependencies"?